include/cuda/std/detail/libcxx/include/atomic

```diff
@@ -1047,10 +1047,15 @@ template<typename _Tp> struct __cxx_is_always_lock_free {
 
 #endif // defined(_LIBCUDACXX_ATOMIC_ALWAYS_LOCK_FREE)
 
-template <typename _Tp, int _Sco,
-          typename _Base = typename conditional<__cxx_is_always_lock_free<_Tp>::__value,
+template <typename _Tp, int _Sco>
+struct __cxx_atomic_impl_conditional {
```

Comment:
Something about `using` is cursed. This is very reminiscent of several `tuple` fixes.

Reply:
DVS results will be posted soon, but this fixed builds on all the compilers I was able to get repros on.